README.md

```diff
@@ -64,6 +64,22 @@ It's in `app/javascript/application.js` you setup your application by importing
 
 It makes sense to use logical names that match the package names used by npm, such that if you later want to start transpiling or bundling your code, you won't have to change any module imports.
 
+### Local modules
+If you want to import local js module files from `app/javascript/src` or other sub-folders of `app/javascript` (such as `channels`), you must pin these to be able to import them.
+You can use `pin_all_from` to pick all files in a specific folder, so you don't have to `pin` each module individually.
+```rb
+#config/importmap.rb
+pin_all_from 'app/javascript/src`, under: 'src', to: 'src'
```

Comment:
If the `:under` option is a suffix of the first parameter, then the `:to` option is not needed. Maybe we can explain the difference between both the options as it was also not clear to me when I first started using the gem.

For more info, you can refer #67.

Reply:
Thanks, sorting now